content/docs/command-reference/remote/add.md

```diff
@@ -482,3 +483,21 @@ $ dvc remote list
 
 myremote	s3://mybucket/another-path
 ```
+
+The credential information can be set via environment variables, for example:
+
+```dvc
+$ export AWS_ACCESS_KEY_ID="<my-access-key>"
+$ export AWS_SECRET_ACCESS_KEY="<my-secret-key>"
+```
+
+or via the dvc (global) config:
+
+```dvc
+$ dvc remote modify --global myremote access_key_id <my-access-key>
+$ dvc remote modify --global myremote secret_access_key <my-secret-key>
+```
+
+You should use this with cautiqon and make sure others don't have access to your
+credential information. For security reason, `--global` is required here because
+you should never commit your credential information to git history.
```

Comment:
Why --global and not --local? I'm still not getting what the scenario we're targeting here is. If someone can explain that to me, then I can prob. give a better review. Thanks

Reply:
IIUC with `--local` the credential information will be written into `<repo>/.dvc/config`, which will be tracked by git. So if there's anyone who doesn't take this carefully, the password might get leaked.

Or perhaps it should leave `dvc` to throw a warning when the user tries to do this?

Reply:
No, with --local another config file will be used, which is put in .gitignore. See https://dvc.org/doc/command-reference/config#command-options-flags. Thanks

Reply:
Oh right, I always thought the default behavior is equivalent to `--local`. Thanks for pointing this out.